contrib/debian/examples/bitcoin.conf

```diff
@@ -95,26 +95,21 @@
 # running on another host using this option:
 #rpcconnect=127.0.0.1
 
-# Transaction Fee Changes in 0.10.0
-
-# Send transactions as zero-fee transactions if possible (default: 0)
-#sendfreetransactions=0
-
 # Create transactions that have enough fees (or priority) so they are likely to begin confirmation within n blocks (default: 1).
 # This setting is over-ridden by the -paytxfee option.
 #txconfirmtarget=n
 
+# Pay an optional transaction fee every time you send bitcoins.  Transactions with fees
+# are more likely than free transactions to be included in generated blocks, so may
+# be validated sooner.
+#paytxfee=0.00
```

Comment:
Oh right. I was just moving the sentence.

A general rewrite should be preferred.